cloudformation.yml

```
@@ -5,6 +5,14 @@ Parameters:
     Description: ECR image URI
     Type: String
 
+  CmrSearchUrl:
+    Description: CMR granule search URL
+    Type: String
```

I'd be more specific here and just ask for a CMR domain from a list of available options. Makes the parameter use more obvious to the reader and reduces their opportunities to make errors.

```suggestion
  CmrDomain:
    Description: CMR granule search URL
    Type: String
    Default: cmr.earthdata.nasa.gov
    AllowedValues:
      - cmr.earthdata.nasa.gov
      - cmr.uat.earthdata.nasa.gov
```